packages/core/scss/components/grid/_layout.scss

```diff
@@ -1277,7 +1277,15 @@
         .k-columnmenu-item:has(> .k-columnmenu-item-actions) {
             display: flex;
             align-items: center;
-            justify-content: space-between;
+            justify-content: center;
+        }
+
+        .k-columnmenu-drag-handle {
+            margin-inline-end: k-spacing(1.5);
```

Comment:
Maybe we could use a variable here?

Reply:
I have the same opinion here, maybe it is not such a good idea to add variables at this moment.